website/versioned_docs/version-0.79/getting-started.md

```diff
@@ -0,0 +1,138 @@
+---
+id: version-0.79-getting-started
+title: Get Started with Windows
+original_id: getting-started
+---
+
+![Architecture](https://img.shields.io/badge/architecture-new_&_old-green)
+
+This guide will help you get started on setting up your very first React Native for Windows app.
+
+Make sure you have installed all of the [development dependencies](rnw-dependencies.md).
+
+For information around how to set up React Native, see the [React Native Getting Started Guide](https://reactnative.dev/docs/getting-started).
+
+> **Interested in migrating to [React Native's New Architecture](https://reactnative.dev/architecture/landing-page)?** New Architecture support in React Native for Windows is now available to preview in 0.76. Note there are several key changes, so if you’d like to be an early adopter, check out the information in the [New Architecture Guide](new-architecture.md). 
+
+## Create a new React Native project
+
+Call the following from the place where you want your project directory to live:
+
+<!-- Note, make sure both `@react-native-community/cli@ABC` and `--version "XYZ"` are pointing to the correct NPM tags in the command below. -->
+
+<!-- 1. For the next version (i.e. in docs/getting-started.md) use "next" for the CLI and "nightly" for the RN version -->
+<!-- 2. For stable versions in versioned_docs use "latest" for the CLI and the semantic version, i.e. "^0.73.0" for the RN version -->
+
+<!-- See https://www.npmjs.com/package/@react-native-community/cli?activeTab=versions for the CLI version tags. -->
+<!-- See https://www.npmjs.com/package/react-native?activeTab=versions for the RN version tags. -->
+
+```bat
+npx --yes @react-native-community/cli@next init <projectName> --version "nightly"
+```
+
+### Navigate into this newly created directory
+
+The command will create your project in a new sub-directory, which you must enter before continuing:
+
+```bat
+cd <projectName>
+```
+
+### Add React Native Windows to your project's dependencies
+
+<!-- Note, make sure "version" is pointing to the correct react-native-windows NPM tag in the command below. -->
+
+<!-- 1. For the next version (i.e. in docs/getting-started.md) use "canary" -->
+<!-- 2. For other versions in versioned_docs use the version in the format "^0.XY.0" -->
+
+Next you'll want to add `react-native-windows` as a dependency:
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Using Yarn (Recommended)-->
+
+```bat
+yarn add react-native-windows@canary
```

Comment:
Updating this should be in the website steps? You need to update the version here with the version we're releasing.

```suggestion
yarn add react-native-windows@^0.79.0
```